auxiliary_tools/metrics_checker.py

```diff
@@ -0,0 +1,60 @@
+#!usr/bin/env python
+from __future__ import division, print_function
+
+import argparse
+import os
+
+"""
+Usage: metrics_checker.py [options]
+Example: python metrics_checker.py -t /lcrc/group/e3sm/public_html/e3sm_diags_test_data/unit_test_complete_run/expected/previous_output/all_sets_v2_6_1_20220328_d62f554/ -r /lcrc/group/e3sm/public_html/e3sm_diags_test_data/unit_test_complete_run/expected/all_sets/
+
+Options:
+  -t FILE, Path to test e3sm_diags results directory
+  -r FILE, Path to reference e3sm_diags results directory
+
+About:
+	This script is used to compare seasonal mean tables between a rest and reference e3sm_diags run,
+        and to print out lines of variables being changed in test.
+"""
+
+
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    "--ref_path",
+    "-r",
+    dest="ref",
+    help="Path to reference e3sm_diags output",
+    metavar="FILE",
+)
+parser.add_argument(
+    "--test_path",
+    "-t",
+    dest="test",
+    help="Path to test e3sm_diags output",
+    metavar="FILE",
+)
+
+args = parser.parse_args()
+
+seasons = ["ANN", "DJF", "MAM", "JJA", "SON"]
+
+ref_path = args.ref
+test_path = args.test
+
+def compare_metrics(ref_path, test_path, season):
+    fref = os.path.join(ref_path, "viewer/table-data", f"{season}_metrics_table.csv")
+    ftest = os.path.join(test_path, "viewer/table-data", f"{season}_metrics_table.csv")
+    try:
+        with open(fref, "r") as ref, open(ftest, "r") as test:
+            file_ref = ref.readlines()
+            file_test = test.readlines()
+            for line in file_ref:
+                if line not in file_test:
+                    print(f"Found difference in {season}", line)
```

Comment:
It might be useful to also check the reverse -- if a line in `file_test` isn't in `file_ref`.

Reply:
This was intentional. If there are additional lines file_test, then there was no reference line to compare with...but it is a good ideal to print the additional variables, I think.

Reply:
Ah ok, that makes sense, but yes it would be nice to know *how* the metrics don't match up rather than just saying a difference exists.